latest/index.bs

```diff
@@ -56,75 +50,66 @@ implementations that are later submitted as a formal specification. (See [[#bf2r
-│
-├── 123.zarr                  # One image (id=123) converted to Zarr.
+├── 123.zarr                  # One OME-Zarr image (id=123).
+│   ...
 │
-└── 456.zarr                  # Another image (id=456) converted to Zarr.
+└── 456.zarr                  # Another OME-Zarr image (id=456).
     │
-    ├── .zgroup               # Each image is a Zarr group, or a folder, of other groups and arrays.
-    ├── .zattrs               # Group level attributes are stored in the .zattrs file and include
-    │                         # "multiscales" and "omero" (see below). In addition, the group level attributes
-    │                         # may also contain "_ARRAY_DIMENSIONS" for compatibility with xarray if this group directly contains multi-scale arrays.
+    ├── zarr.json             # Each image is a Zarr group of other groups and arrays.
+    │                         # Group level attributes are stored in the `zarr.json` file and include
+    │                         # "multiscales" and "omero" (see below).
     │
     ├── 0                     # Each multiscale level is stored as a separate Zarr array,
     │   ...                   # which is a folder containing chunk files which compose the array.
     ├── n                     # The name of the array is arbitrary with the ordering defined by
     │   │                     # by the "multiscales" metadata, but is often a sequence starting at 0.
     │   │
-    │   ├── .zarray           # All image arrays must be up to 5-dimensional
+    │   ├── zarr.json         # All image arrays must be up to 5-dimensional
     │   │                     # with the axis of type time before type channel, before spatial axes.
     │   │
-    │   └─ t                  # Chunks are stored with the nested directory layout.
-    │      └─ c               # All but the last chunk element are stored as directories.
-    │         └─ z            # The terminal chunk is a file. Together the directory and file names
-    │            └─ y         # provide the "chunk coordinate" (t, c, z, y, x), where the maximum coordinate
-    │               └─ x      # will be `dimension_size / chunk_size`.
+    │   └─ ...                # Chunks are stored conforming to the Zarr array specification and 
+    │                         # metadata as specified in the array's `zarr.json`.
     │
     └── labels
         │
-        ├── .zgroup           # The labels group is a container which holds a list of labels to make the objects easily discoverable
-        │
-        ├── .zattrs           # All labels will be listed in `.zattrs` e.g. `{ "labels": [ "original/0" ] }`
-        │                     # Each dimension of the label `(t, c, z, y, x)` should be either the same as the
+        ├── zarr.json         # The labels group is a container which holds a list of labels to make the objects easily discoverable
+        │                     # All labels will be listed in `zarr.json` e.g. `{ "labels": [ "original/0" ] }`
```

Comment:
Previously, the `.zattrs` in it's entirety was e.g. `{ "labels": [ "original/0" ] }` but now the `zarr.json` will be e.g:

```
{
    "zarr_format": 3,
    "node_type": "group",
    "attributes": {
        "labels": [ "original/0" ]
    }
}
```
Is that right? If so, this is probably too verbose to include in-line here, but maybe some rewording could describe/summarise that? E.g. 

All labels will be listed within `zarr.json` e.g. `"attributes": { "labels": [ "original/0" ] }`

Reply:
Ah, I realise this will actually need `"attributes": {"https://ngff.openmicroscopy.org/0.5": { "labels": [ "original/0" ] }}`